public/static/app/perfdiscovery/PerformanceDiscoveryCtrl.js

```diff
@@ -92,6 +92,39 @@ mciModule.controller('PerformanceDiscoveryCtrl', function (
     vm.updateData(true);
```

Comment:
it looks like reload is no longer needed, although it's difficult to tell without being able to run the code.

Reply:
Reload is still called when you change the data source